### ISSUE TITLE
Type of imported str literal incorrectly becoming str in some context

### ISSUE THREAD

**Issue by giyyapan** [microsoft/pyright#528]
**Describe the bug**
Imported str literal type is becoming str in some special context, please see this example.

![image](https://user-images.githubusercontent.com/3479453/74720275-7cb0d580-5270-11ea-8db5-4b0c5308e31b.png)

I've try to reproduce this bug in a simpler demo, but it seems that the bug will only occur when these conditions are fulfilled:
1. The literal must be imported from another file.
2. The literal must be used in a class method body. 

Let me know if you need more information : )

**Code**
In file test.py
```python
from .test2 import T

class A:
    def some_method(self):
        reveal_type(T) # type of T is incorrect


reveal_type(T) # type of T is correct
```
In file test2.py
```python
from typing_extensions import Literal

T = Literal['a', 'b', 'c']
```

**VS Code extension or command-line**
I'm using VS Code extension 1.1.22 and Python3.7 with virtualenv

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of Pyright.

**Comment by erictraut** [comment]
This is now fixed in 1.1.24, which I just published.
